nora/src/content/browser-splitView/browser-splitView.pcss

```diff
@@ -0,0 +1,41 @@
+
+
+.deck-selected {
+  display: flex !important;
+  flex: auto !important;
+  order: 2;
+
+  &[splitview="right"] {
+    order: 4 !important;
+  }
+
+  &[splitview="left"] {
+    order: 0 !important;
+  }
+}
+
+#splitview-splitter {
+  flex: 0 !important;
+  border: none;
+}
+
+#tabbrowser-tabpanels {
+  display: flex !important;
+
+  > * {
```

Comment:
it seems works but `& > *` is more readable I think
when using css nesting, you will have many times to use &
so when using `>` or `<`, I recommend to use with `&`

reference
https://www.w3.org/TR/css-nesting-1/#example-31845edf
adding & seems common because in this example, it appears as one of the valid thing

Reply:
Fixed